integration_test/docker/test-docker-compose.yml

```diff
@@ -3,7 +3,7 @@ version: '3'
 services:
   core0:
     container_name: core0
-    image: "ghcr.io/celestiaorg/celestia-app:v0.10.0-rc1"
+    image: "ghcr.io/celestiaorg/celestia-app:v0.10.0"
```

Comment:
Regarding Issue #37 Should this be Version 10 or 11 ?

Reply:
works with 0.10.0, error 13 with 0.11.0 , so I changed Title

Reply:
We need v0.11.0 compatibility, as this is version used by Arabica testnet.